changelogs/unreleased/1392-anshulc

```diff
@@ -0,0 +1 @@
+Validate restore name label length
```

Comment:
Can you update this to something like "shorten label values when they're longer than 63 characters"?

Reply:
Done